apps/acrs/translations/src/en/fields.json

```diff
@@ -20,15 +20,8 @@
     }
   },
   "full-name": {
-      "label": "SKELETON: full-name",
-      "options": {
-        "under-18": {
-          "label": "SKELETON: Referrer DOB < 28-09-2023 (under 18)"
-        },
-        "over-18": {
-          "label": "SKELETON: Referrer DOB >= 28-09-2023 (over 18)"
-        }
-      }
+    "label": " ",
```

Comment:
You will need to use the text of the question as the label for accessibility purposes here. It should then show up as the H1 if you add `isPageHeading: true` to the relevant block in acrs/fields/index.js.

Reply:
Good to know 👍